src/components/table/LightTable.tsx

```diff
@@ -309,52 +300,51 @@ function LightTable({
               />
               <TableBody data-cy="tableBody">
                 {rows && rows.length > 0 ? (
-                  rows.map((row: any, index) => {
+                  rows.map((row: any, index: number) => {
                     const darkRow = index % 2 !== 0;
                     return (
-                      <TableRow
+                      <Row
                         hover
                         key={`${index}-${row[rowKey || 'id']}}`}
-                        className={clsx({
-                          [classes.row]: true,
-                          [classes.darkRow]: darkRow,
-                          [classes.lightRow]: !darkRow,
-                        })}
+                        sx={{
+                          backgroundColor: darkRow
+                            ? theme.palette.lightTable.cellBackgroundDark
+                            : theme.palette.lightTable.callBackgroundLight,
```

Comment:
```suggestion
                            ? lightTable.cellBackgroundDark
                            : lightTable.callBackgroundLight,
```

Reply:
This wouldn't work - it is accessing the property and needs the full path. Unless I could just use the string `lightTable.cellBackgroundDark`?